reframe_config.py

```diff
@@ -107,6 +111,12 @@
             'cxx': 'g++',
             'ftn': 'gfortran'
         },
+        {
+            'name': 'intel',
+            'cc': 'icc',
+            'cxx': 'icpc',
+            'ftn': 'ifort'
+        },
```

Comment:
I think we can also remove the programming environment altogether.  Spack is not going to consider it anyways.

Reply:
Why don't we just use the`builtin` environment if we never use any compilers?

Reply:
I seem to remember trying to remove all environments from here and getting an error because at least one programming environment must be specified, and that's why I had left the "gnu" one above, not because it was needed for something specific.  Is there a way to specify a builtin environment?  I hadn't explored that